Missing module error with the babelification

@floatdrop after 1d5ef4c5f6fcb70f9e90e584dfd08865db1e93b4 landed I'm now getting this error when testing `got` using master.

```
~/dev/got master
❯ ava

module.js:339
    throw err;
    ^

Error: Cannot find module 'babel-runtime/helpers/async-to-generator'
    at Function.Module._resolveFilename (module.js:337:15)
    at Function.Module._load (module.js:287:25)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at Object.<anonymous> (/Users/sindresorhus/dev/got/test/unix-socket.js:3:25)
    at Module._compile (module.js:435:26)
    at requireFromString (/Users/sindresorhus/dev/ava/node_modules/require-from-string/index.js:14:4)
    at Object.<anonymous> (/Users/sindresorhus/dev/ava/lib/babel.js:28:1)
    at Module._compile (module.js:435:26)
    at Object.Module._extensions..js (module.js:442:10)
module.js:339
    throw err;
    ^

Error: Cannot find module 'babel-runtime/helpers/async-to-generator'
    at Function.Module._resolveFilename (module.js:337:15)
    at Function.Module._load (module.js:287:25)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at Object.<anonymous> (/Users/sindresorhus/dev/got/test/retry.js:3:25)
    at Module._compile (module.js:435:26)
    at requireFromString (/Users/sindresorhus/dev/ava/node_modules/require-from-string/index.js:14:4)
    at Object.<anonymous> (/Users/sindresorhus/dev/ava/lib/babel.js:28:1)
    at Module._compile (module.js:435:26)
    at Object.Module._extensions..js (module.js:442:10)
TypeError: Cannot read property 'stack' of undefined
    at error (/Users/sindresorhus/dev/ava/cli.js:51:19)
From previous event:
    at Object.<anonymous> (/Users/sindresorhus/dev/ava/cli.js:218:34)
    at Module._compile (module.js:435:26)
    at Object.Module._extensions..js (module.js:442:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:311:12)
    at Function.Module.runMain (module.js:467:10)
    at startup (node.js:134:18)
    at node.js:961:3
```

--- a/lib/babel.js
+++ b/lib/babel.js
@@ -2,9 +2,9 @@
 var resolveFrom = require('resolve-from');
 var createEspowerPlugin = require('babel-plugin-espower/create');
 var requireFromString = require('require-from-string');
-var hasGenerators = parseInt(process.version.slice(1), 10) > 0;
-var path = process.argv[2];
 
+var hasGenerators = parseInt(process.version.slice(1), 10) > 0;
+var testPath = process.argv[2];
 var babel;
 
 try {
@@ -24,5 +24,7 @@ var options = {
 	]
 };
 
-var transpiled = babel.transformFileSync(path, options);
-requireFromString(transpiled.code, path);
+var transpiled = babel.transformFileSync(testPath, options);
+requireFromString(transpiled.code, testPath, {
+	appendPaths: module.paths
+});

--- a/package.json
+++ b/package.json
@@ -80,7 +80,7 @@
     "power-assert-formatter": "^1.3.0",
     "power-assert-renderers": "^0.1.0",
     "pretty-ms": "^2.0.0",
-    "require-from-string": "^1.0.0",
+    "require-from-string": "^1.1.0",
     "resolve-from": "^1.0.0",
     "serialize-error": "^1.0.0",
     "set-immediate-shim": "^1.0.1",

